simd/src/Kokkos_SIMD_NEON.hpp

```diff
@@ -363,10 +363,18 @@ class simd<double, simd_abi::neon_fixed_size<2>> {
                                                        element_aligned_tag) {
     m_value = vld1q_f64(ptr);
   }
+  KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION void copy_from(value_type const* ptr,
+                                                       vector_aligned_tag) {
+    m_value = vld1q_f64(ptr);
```

Comment:
So no specializations for NEON?

Reply:
It doesn't look like a separate intrinsic is provided for aligned loads/stores for NEON.
NEON documentation suggests that an alignment hint can be provided for [ISA](https://developer.arm.com/documentation/den0018/a/NEON-Instruction-Set-Architecture/Alignment) but nothing about it in [intrinsics](https://developer.arm.com/documentation/den0018/a/NEON-Intrinsics/Loading-data-from-memory-into-vectors?lang=en).